src/cardLikes.js

```diff
@@ -0,0 +1,45 @@
+const URL = 'https://us-central1-involvement-api.cloudfunctions.net/capstoneApi/apps/cOGEfAnTEwb10Y4bv0g7/likes/';
```

Comment:
- [x] I would recommend separating the **API code** from the URL, that way you can change it more easily.

```suggestion
const apiCode = 'cOGEfAnTEwb10Y4bv0g7';
const URL = `https://us-central1-involvement-api.cloudfunctions.net/capstoneApi/apps/${apiCode}/likes/`;

```